src/data/int/modeq.lean

```diff
@@ -28,6 +28,11 @@ def modeq (n a b : ℤ) := a % n = b % n
 
 notation a ` ≡ `:50 b ` [ZMOD `:50 n `]`:0 := modeq n a b
 
+notation a ` ≢ `:50 b ` [ZMOD `:50 n `]`:0 := ¬ (a ≡ b [ZMOD n])
```

Comment:
The U+2262 `≢` character seems to have very poor font support (some discussion at https://github.com/harfbuzz/harfbuzz/issues/1092), so I'm not convinced this notation is a good idea.

Reply:
Feel free to close to PR if you think is not a good idea.